packages/assistify-threading/lib/threadRoomType.js

```diff
@@ -0,0 +1,19 @@
+import { RocketChat, RoomTypeConfig } from 'meteor/rocketchat:lib';
+
+export class ThreadRoomType extends RoomTypeConfig {
+	constructor() {
+		super({
+			identifier: 'thread',
+			order: 25,
+			label: 'Threads',
+		});
+
+		this.customTemplate = 'ThreadList';
+	}
+
+	condition() {
+		return RocketChat.getUserPreference(Meteor.userId(), 'sidebarShowThreads');
```

Comment:
Should we not check the user's permissions here? :thinking:  Because the "create thread button" keeps showing up even with the permission disabled.

Reply:
Yes, we should!

Reply:
Added validation on authorization - but in another file ;)